package.json

```diff
@@ -79,6 +82,6 @@
     "eslint-plugin-react": "^7.19.0",
     "http-proxy-middleware": "^1.0.5",
     "react-axe": "^3.4.1",
-    "react-hook-form-devtools": "^1.2.0"
```

Comment:
The new version of this doesn't seem to exist in npm
When I put it back to the old version it updated package-lock.json as it hadn't been there....
@PanopticaRising can you please let me know what the correct action is?